pkg/services/ngalert/sender/router.go

```diff
@@ -150,7 +149,7 @@ func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
 				continue
 			}
 
-			d.logger.Debug("applying new configuration to sender", "org", cfg.OrgID, "alertmanagers", redactedAMs)
+			d.logger.Info("applying new configuration to sender", "org", cfg.OrgID, "alertmanagers", redactedAMs)
```

Comment:
Do we have the configuration ID here? If yes we could add it to the context.

Reply:
Good idea, done